packages/mesos-metrics-module/build

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+mkdir -p build
+pushd build
+
+# configure module build with cmake
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DTESTS_ENABLED=False \
+  -DCMAKE_INSTALL_PREFIX=$PKG_PATH \
+  -Dmesos_INCLUDE_DIR=/opt/mesosphere/include \
+  -Ddefault_BIN_DIR=/opt/mesosphere/bin \
+  -Ddefault_LIB_DIR=/opt/mesosphere/lib \
+  -DUSE_LOCAL_PROTOBUF=False \
+  -Dprotobuf_INCLUDE_DIR=/opt/mesosphere/lib/mesos/3rdparty/include \
+  -DUSE_LOCAL_GLOG=False \
+  -Dglog_INCLUDE_DIR=/opt/mesosphere/lib/mesos/3rdparty/include \
+  -DUSE_LOCAL_AVRO=False \
+  /pkg/src/mesos-metrics-module/mesos_module/
+
+make -j8
```

Comment:
This needs to use `make -j$NUM_CORES` (see: https://github.com/dcos/dcos/commit/c55b0c82c3c74d1e5c37b7d944d527bcda6e5dd9)